src/pocketmine/level/Level.php

```diff
@@ -1222,12 +1222,9 @@ public function rayTraceBlocks(Vector3 $pos1, Vector3 $pos2, $flag = false, $fla
 	public function getFullLight(Vector3 $pos) : int{
 		$chunk = $this->getChunk($pos->x >> 4, $pos->z >> 4, false);
 		$level = 0;
-		if($chunk !== null){
-			$level = $chunk->getBlockSkyLight($pos->x & 0x0f, $pos->y & Level::Y_MASK, $pos->z & 0x0f);
-			//TODO: decrease light level by time of day
-			if($level < 15){
-				$level = max($chunk->getBlockLight($pos->x & 0x0f, $pos->y & Level::Y_MASK, $pos->z & 0x0f));
-			}
+		if($chunk instanceof Chunk){
+			//Do not decrease skylightlvl over day. MCPC and MCPE don't do this either.
+			$level = max($chunk->getBlockLight($pos->x & 0x0f, $pos->y & Level::Y_MASK, $pos->z & 0x0f), $chunk->getBlockSkyLight($pos->x & 0x0f, $pos->y & Level::Y_MASK, $pos->z & 0x0f));
```

Comment:
The Y mask is not needed, and I don't recommend using it, as it will produce unexpected wraparound values.